### DISCUSSION TITLE
Disable Flux from managing dark mode solution

### DISCUSSION THREAD

**Discussion by igmltd** [livewire/flux#913] [Feature Requests]
Hi All,

With the introduction of flux night mode feature, although great, myself and others across multiple threads wanted a way to tell flux **"Let me manage dark mode"**.

For those of you who want to manage it, just add the following to your tailwind config

`darkMode: ['selector', '.night']`

`.night` can be changed to what ever you want, just means tailwind will look for that rather than `.dark` in the body.

@calebporzio done a great job to make it just work, however I personally found it a little app intrusive where the site was just changing to dark mode -  many just change tailwind to 'selector' to just turn off dark mode and there are some situations where you just want to manage it.

It would have probably been an idea to have it the other way round, where you add `.flux-dark` to the tailwind config,  to then tell flux to explicitly manage it, but this works.

Hope this helps others.

Keep Fluxin!

**Comment by joffreypersia** [comment]
Yes, it was my case because we also store in the database the state of the user preference.
